docs/docs/getting-started-digitalocean/index.md

```diff
@@ -0,0 +1,110 @@
+# Getting Started with DigitalOcean
+
+DigitalOcean is a very simple provider to get set up. This guide will take you through the whole process.
+
+1. **[Sign Up](#sign-up)**
+2. **[Create your Personal Access Token](#create-pat)**
+3. **[Create your Stack](#create-stack)**
+4. **[Set your Credentials](#set-creds)**
+5. **[Save your Stack and Initialize](#save-init)**
+
+***
+
+## Sign Up <a name="sign-up"></a>
+
+Using DigitalOcean as your provider will require you to sign up on their website. It will also require you to enter your credit card information and make an initial payment of at least $5. You can do that [here](https://cloud.digitalocean.com/registrations/new). 
+
+[Learn more about pricing](https://www.digitalocean.com/pricing/) at DigitalOcean.
+
+***
+
+## Create your Personal Access Token <a name="create-pat"></a>
+
+Your DigitalOcean Personal Access Token will provide the credentials you need to instantiate your Stacks using DigitalOcean. 
+
+To generate your token:
+
+1. [Login](https://cloud.digitalocean.com/login) to Digital Ocean
+2. [Click API](https://cloud.digitalocean.com/settings/api/tokens) from the top menu
+3. On the Tokens tab, click [Generate New Token](https://cloud.digitalocean.com/settings/api/tokens/new)
+4. Name your new Token
+5. Give it Read and Write permissions
+6. Click Generate Token
+
+> You'll use this token string as your credentials for creating VMs with Koding later in this guide. 
+
+***
+
+## Create your Stack <a name="create-stack"></a>
+
+To create your Stack:
+
+1. [Login to Koding](https://koding.com/Teams/Select)
+2. Go to your [Dashboard](https://relepic.koding.com/Home) OR click [Stacks](https://relepic.koding.com/Home/Stacks) from your sidebar.
+3. Either way, you'll default to the Stacks screen, so from that screen click [New Stack](https://relepic.koding.com/Stack-Editor/New)
+4. Use the Wizard to create and  populate your Stack with some defaults, make sure you select Digital Ocean as provider
```

Comment:
Make selecting Digital Ocean a specific step and then say they can optionally skip the remaining steps or continue to install some default software.